setup.py

```diff
@@ -60,16 +60,20 @@
     'Programming Language :: Python',
     'Programming Language :: Python :: 2',
     'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
     'Programming Language :: Python :: Implementation :: CPython',
     'Topic :: Security',
     'Topic :: Software Development'
   ],
-  # securesystemslib 0.10.8 requires cryptography>=2.1.3, and thereby dictates
-  # the minimum version of cryptography for in-toto. The maximum version
-  # is dictated by what is available.
   packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests",
       "debian"]),
-  install_requires=["six", "cryptography", "securesystemslib==0.10.8", "attrs",
+  # securesystemslib 0.10.9 requires cryptography>=2.1.3, and thereby dictates
```

Comment:
Note: `cryptography` can, and occasionally does, introduce backwards-incompatible changes.  It is often for security and API reasons.  See their [API stability](https://cryptography.io/en/latest/api-stability/) and [changelog history](https://cryptography.io/en/latest/changelog/) docs.

I recommend that you periodically check for any breaking changes introduced by new `cryptography` releases, unless your Travis CI/pyup.io bots already do so.

Reply:
Since some of our unittests verify pre-signed test metadata they should break if we start installing a backwards-incompatible versions of cryptography. I also have securesystemslib's pyup bots on the radar. And in-toto's gpg code (which require cryptography) should be factored out to securesystemslib altogether at some point anyway. :)